weblate/templates/dictionaries.html

```diff
@@ -1,13 +1,108 @@
+        <li><a href="{% url 'checks' %}?project={{ object.slug }}">{% trans "Failing checks" %}</a></li>
+        <li><a href="{% url 'checks' %}?project={{ object.slug }}&amp;ignored=true">{% trans "Ignored checks" %}</a></li>
+      </ul>
+    </li>
+    {% if user_can_manage_acl or user_can_edit_project or user_can_see_repository_status or user.is_authenticated %}
+    <li class="dropdown">
+      <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+        {% trans "Manage" %} <span class="caret"></span>
+      </a>
+      <ul class="dropdown-menu">
+        {% if user_can_commit_translation %}
+        <li><a href="{{ object.get_commit_url }}" class="link-post">{% trans "Commit" %}</a></li>
+        {% endif %}
+        {% if user_can_see_repository_status %}
+        <li><a href="{% url 'project' project=object.slug %}#repository" data-href="{% url 'git_status_project' project=object.slug %}">{% trans "Repository maintenance" %}</a></li>
+        <li role="separator" class="divider"></li>
+        {% endif %}
+        <li><a href="{% url 'memory' project=object.slug %}">{% trans "Translation memory" %}</a></li>
+        {% if whiteboard_form %}
+        <li><a href="{% url 'project' project=object.slug %}#whiteboard">{% trans "Add a project message" %}</a></li>
+        {% endif %}
+        {% if user_can_manage_acl %}
+        <li><a href="{% url 'manage-access' project=object.slug %}">{% trans "Users" %}</a></li>
+        {% endif %}
+        {% if user_can_edit_project %}
+        <li><a href="{% url 'fonts' project=object.slug %}">{% trans "Fonts" %}</a></li>
+        <li><a href="{% url 'settings' project=object.slug %}">{% trans "Settings" %}</a></li>
+        {% endif %}
+        {% if delete_form or rename_form %}
+        <li role="separator" class="divider"></li>
+        <li><a href="{% url 'project' project=object.slug %}#rename">{% trans "Rename" %}</a></li>
+        <li><a href="{% url 'project' project=object.slug %}#delete">{% trans "Removal" %}</a></li>
+        {% endif %}
+      </ul>
+    </li>
+    {% endif %}
+  {% if enable_sharing %}
+    <li class="dropdown">
+      <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+        {% trans "Share" %} <span class="caret"></span>
+      </a>
+      <ul class="dropdown-menu">
+  {% with object.get_share_url as share_url and object.get_widgets_url as widgets_url %}
+        <li><a href="https://www.facebook.com/sharer.php?u={{ share_url }}">{% trans "Share on Facebook" %}</a></li>
+        <li><a href="https://twitter.com/share?text={% blocktrans %}Translate {{ object }} using %23Weblate at {{ share_url }}!{% endblocktrans %}">{% trans "Tweet this translation" %}</a></li>
+        <li><a href="{% url 'engage' project=object.slug %}">{% trans "Engage page" %}</a></li>
+        <li><a href="{{ widgets_url }}">{% trans "Status widgets" %}</a></li>
+  {% endwith %}
+        </ul>
+    </li>
+  {% endif %}
+</ul>
+
```

Comment:
I'd really prefer if the menu would be in separate template and included in both pages. Copying it will make it easy to miss updates in one of these.

Reply:
Sure thing, I'm on it :)